patch/0022-dts-marvell-Add-DB-CN913X-Compexpress-board.patch

```diff
@@ -0,0 +1,560 @@
+From: Pavan Naregundi <pnaregundi@marvell.com>
+Date: Thu, 14 Sep 2023 04:35:53 +0000
+dts: marvell: Add DB-CN913X-Compexpress board
+
+This change adds device tree file for Marvell CN913x ComExpress
+development board.
+
+The DB-CN913X-ComExpress Card is a basic Type 7 module, aimed to serve
+as a system management for the Switch family boards.
+
+The CN9130 composed of:
+ - AP807 consisting of Quad Core ARMv8 Cortex-A72 and few peripherals.
+ - One CP115 block that contain all the high-speed interfaces
+   (PCIe, Ethernet, etc.).
+
+CN9131 is comprised from a single CN9130 with additional CP115 block.
+
```

Comment:
Please add the URL, where you have sent this upstream for review/inclusion.

Reply:
Currently we have cn913x-db.dts and cn913x-crb.dts up streamed in Linux kernel. This board(cn913x-db-comexpress.dts) is a variant of cn913x-crb.dts. If upstream is still needed, we need push them for upstream review.